test/pjrt/test_experimental_pjrt.py

```diff
@@ -0,0 +1,88 @@
+import concurrent.futures
+import os
+import time
+
+import torch
+import torch_xla
+from absl.testing import absltest, parameterized
+import torch_xla.core.xla_env_vars as xenv
+import torch_xla.core.xla_model as xm
+from torch_xla.experimental import pjrt
+
+
+def _mp_fn():
+  """Pickle-able function to run multiprocess."""
+  return xm.xla_device()
+
+
+class TestExperimentalPjrt(parameterized.TestCase):
+
+  def setUp(self):
+    pjrt.set_device_type('CPU')
+
+  def test_using_pjrt(self):
+    del os.environ[xenv.PJRT_DEVICE]
+
+    self.assertFalse(pjrt.using_pjrt())
+
+  def test_requires_pjrt(self):
+    del os.environ[xenv.PJRT_DEVICE]
+
+    with self.assertRaises(NotImplementedError):
+      pjrt.xla_device()
+
+  def test_default_ordinals(self):
+    global_ordinal = xm.get_ordinal()
+    self.assertEqual(global_ordinal, 0)
+
+    local_ordinal = xm.get_local_ordinal()
+    self.assertEqual(local_ordinal, 0)
+
+  def test_num_local_devices(self):
```

Comment:
`get_supported_devices()` returns the set of devices that are visible to the current process. Each thread in the process can "see" all of the addressable devices because they share the same client.